searxng_extra/update/update_engine_descriptions.py

```diff
@@ -206,7 +206,7 @@ def initialize():
 
 def fetch_wikidata_descriptions():
     print('Fetching wikidata descriptions')
-    searx.network.set_timeout_for_thread(60)
+    searx.network.set_networkcontext_for_thread(None, 60.0)
```

Comment:
There is no need to specify a network context time out:

`searx.network.set_networkcontext_for_thread(None, None)`

or

`searx.network.set_networkcontext_for_thread()`

?